sql-files/mob_db.sql

```diff
@@ -62,6 +62,7 @@ CREATE TABLE `mob_db` (
   `racegroup_ep172bath` tinyint(1) unsigned DEFAULT NULL,
   `racegroup_illusion_turtle` tinyint(1) unsigned DEFAULT NULL,
   `racegroup_rachel_sanctuary` tinyint(1) unsigned DEFAULT NULL,
+  `racegroup_Illusion_Luanda` tinyint(1) unsigned DEFAULT NULL,
```

Comment:
```suggestion
  `racegroup_illusion_luanda` tinyint(1) unsigned DEFAULT NULL,
```
Isn't it better to follow the standard and leave all "racegroup_Illusion_Luanda" with lowercase letters?

Reply:
Thanks!